server_structs/director.go

```diff
@@ -76,6 +76,7 @@ type (
 		Latitude     float64    `json:"latitude"`
 		Longitude    float64    `json:"longitude"`
 		Writes       bool       `json:"enable_write"`
+		Listings     bool       `json:"Listing"`              // True if the origin allows directory listings
```

Comment:
Match the style of the other flags
```suggestion
		Listings     bool       `json:"enable_listing"`              // True if the origin allows directory listings
```